single-post.php

```diff
@@ -0,0 +1,16 @@
+<?php get_header(); the_post(); ?>
+
+<div class="featured-image">
+	<?php the_post_thumbnail() ?>
```

Comment:
You can add classes to a featured image by passing them in the arguments https://developer.wordpress.org/reference/functions/get_the_post_thumbnail/.

As an example:

```suggestion
	<?php the_post_thumbnail( 'large', array( 'class' => 'img-fluid text-center object-fit-cover' ) ); ?>
```

Reply:
CSS was added via class and since the_post_thumbnail accepts array, I have added the following
	<?php the_post_thumbnail( 'large', array( 'class' => 'w-100 img-fluid text-center object-fit-cover' , 'style' => 'height: 500px') ) ?>